playground/nextjs-app-router/components/Demo.tsx

```diff
@@ -36,7 +58,7 @@ function Demo() {
           View Github
         </a>
       </div>
-      <div className="flex flex-1 flex-col bg-[linear-gradient(to_right,#f0f0f0_1px,transparent_1px),linear-gradient(to_bottom,#f0f0f0_1px,transparent_1px)] bg-[size:6rem_4rem]">
+      <div className="linear-gradient(to_bottom,#f0f0f0_1px,transparent_1px)] flex flex-1 flex-col bg-[linear-gradient(to_right,#f0f0f0_1px,transparent_1px), bg-[size:6rem_4rem]">
```

Comment:
Wanted to cut this into multiple lines but the linter is preventing it.

Reply:
use `cn` of `clx` for this